vnext/Shared/RuntimeOptions.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <string>
+
+namespace Microsoft::React {
+
+/// <summary>
+/// Sets a global boolean value identified by an arbitrary string.
+/// </summary>
+/// <param name="name">Global key</param>
+void __cdecl SetRuntimeOptionBool(std::string &&name, bool value) noexcept;
```

Comment:
>SetRuntimeOptionBool [](start = 13, length = 20)

(nit) maybe at some point we would also want to add "RemoveRuntimeOption" Api to have a complete story here, but that's for the future.

Reply:
I'd go for `Clear` or `Unset` instead of `Remove`, but I'll add those as needed. For now, we could just set them to `0` or `false` to achieve the same effect (not as performant as removing the key, of course).

It's possible by the time we would need those functions, the Desktop and Universal stories will have converged and these APIs will be dropped in favor of the ones available since 0.63.